src/lib/paths/index.js

```diff
@@ -18,7 +18,7 @@ export function auroraPreferencesFile() {
 
 // ~/.aurora/<NODE_ENV>/mutations
 export function auroraMutationPath() {
-  return path.join(auroraUserPath(), "mutations");
+  return path.join(auroraUserPath(), "extensions");
```

Comment:
Not sure we want to change this path. This is where we store mutations. Don't change this.

Reply:
I mean it's fine if it doesn't break anything